responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -257,12 +257,16 @@ def get_data(self):
             self._get_counterfactual(i) for i in self.get()]
 
     def _get_counterfactual(self, counterfactual):
-        counterfactual_data = CounterfactualData()
+        cfdata = CounterfactualData()
         json_data = json.loads(counterfactual.to_json())
-        counterfactual_data.cfsList = json_data["cfs_list"]
-        counterfactual_data.featureNames = json_data[
-            "feature_names_including_target"]
```

Comment:
For the counterfactual in cf_list, the cf_list includes the target. You would require feature_names_including_target for cf_list.